### PR TITLE
SmallerDegreePerm... renamed, and now an attribute

### PR DESCRIPTION
**Pull request by cdwensley**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,7 +1,9 @@
 # CHANGES to the 'XMod' package
 
-## 2.77 -> 2.77dev (06/03/2020) 
+## 2.77 -> 2.77dev (10/03/2020) 
 
+ * (10/03/20) now SmallerDegreePermutationRepresentation2DimensionalGroup in
+              place of SmallerDegreePerm2DimensionalDomain
  * (06/03/20) now running under GAP 4.11.0
```
Comment (alex-konovalov) [inline CHANGES.md]: Probably **requiring** at least GAP 4.11.0 is an essential change.

File: PackageInfo.g [hunk 2]
```diff
@@ -104,6 +104,8 @@ PackageDoc := rec(
   Autoload  := true
 ),
 
+SetInfoLevel(InfoPackageLoading,4);
```
Comment (alex-konovalov) [inline PackageInfo.g]: This does not belong here - you can not plug the GAP command inside a record. Also, at this point loading package already started. Try to put it into tst/testall.g before LoadPackage( "xmod" );

Comment (cdwensley) [reply PackageInfo.g]: This was just a trial - I was not surprised when it did not work.

**Comment by alex-konovalov** [comment]
@cdwensley tests failing - you need to rebase this branch on master:

    git rebase master
    git push -f

will hopefully work.

**Pull request merged**